src/ddmd/builtin.d

```diff
@@ -216,6 +216,37 @@ extern (C++) Expression eval_yl2xp1(Loc loc, FuncDeclaration fd, Expressions* ar
     return new RealExp(loc, result, arg0.type);
 }
 
+extern (C++) Expression builtin_ctfeWrite(Loc loc, FuncDeclaration fd, Expressions* arguments)
+{
+    import core.stdc.stdio : fprintf;
+    import ddmd.ctfeexpr : resolveSlice;
+
+    Expression arg0 = (*arguments)[0];
+    StringExp se;
+
+    switch (arg0.op)
+    {
+        case TOKstring:
+            se = cast(StringExp) arg0;
+            break;
+        case TOKslice:
+            se = resolveSlice(arg0).toStringExp();
+            break;
+        case TOKarrayliteral:
+            se = arg0.toStringExp();
```

Comment:
The success check is in line 242 (same as with the above case), as failure means `se` remains `null`.